samples/styles/reslist.css

```diff
@@ -0,0 +1,4 @@
+
+select option {
```

Comment:
Firefox `<option>` inherits whatever styles are on the `<select>` which ... is odd.

Reply:
I need to start doing UI screenshots between browsers because Safari does weird things too with selects.